site/pages/docs/ref/provisional-en.hbs

```diff
@@ -111,3 +117,18 @@
 		&lt;p&gt;Secondary title&lt;/p&gt;
 	&lt;/div&gt;
 &lt;/div&gt;</code></pre>
+
+<h3 id="well-header-rwd-bgimg">Showing random image using [data-wb-randomize] plugin</h3>
+<p>Below example shows one image out of three images randomly. Refresh the page to see that the image changes</p>
+<div class="provisional" data-wb-randomize="toggle@self!img@{&quot;number&quot;:1,&quot;classes&quot;: &quot;hide&quot;}">
```

Comment:
Can you review how the content in the attribute "data-wb-randomize" is structured. In WET4 we have either one single value or a JSON object that contains multiple values.